Paintroid/src/main/java/org/catrobat/paintroid/tools/helper/Conversion.kt

```diff
@@ -16,18 +16,18 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+package org.catrobat.paintroid.tools.helper
 
-package org.catrobat.paintroid.ui.tools;
+import android.graphics.Point
+import android.graphics.PointF
 
-import android.text.InputFilter;
-import android.text.Spanned;
+class Conversion private constructor() {
```

Comment:
Please check the git branch for possible faults.
NumberRangeFilter.java -> Conversions.kt

Reply:
github bug